From 6dd3af00e508aeaecbe8e8e92b92725a2d31297c Mon Sep 17 00:00:00 2001
From: Bhaskar Upadhaya <Bhaskar.Upadhaya@nxp.com>
Date: Thu, 7 Dec 2017 07:34:55 +0530
Subject: [PATCH 208/211] ls1012a:dts: Correct LS1012A-2G5RDB device tree

Previously LS1012A-2G5RDB dts includes fsl-ls1012a-rdb.dtsi which
further includes fsl-ls1012a.dtsi.Now LS1012A-2G5RDB dts include
fsl-ls1012a.dtsi.

Signed-off-by: Bhaskar Upadhaya <Bhaskar.Upadhaya@nxp.com>
---
 arch/arm/dts/fsl-ls1012a-2g5rdb.dts |   31 +++++++++++++++++++++++++++++--
 1 files changed, 29 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/fsl-ls1012a-2g5rdb.dts b/arch/arm/dts/fsl-ls1012a-2g5rdb.dts
index fdb37b4..5a45c3c 100644
--- a/arch/arm/dts/fsl-ls1012a-2g5rdb.dts
+++ b/arch/arm/dts/fsl-ls1012a-2g5rdb.dts
@@ -7,10 +7,37 @@
  */
 
 /dts-v1/;
-#include "fsl-ls1012a-rdb.dtsi"
+#include "fsl-ls1012a.dtsi"
 
 / {
-       chosen {
+	model = "LS1012A 2G5RDB Board";
+
+	aliases {
+		spi0 = &qspi;
+	};
+
+	chosen {
                stdout-path = &duart0;
        };
 };
+
+&qspi {
+	bus-num = <0>;
+	status = "okay";
+
+	qflash0: s25fl128s@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "spi-flash";
+		spi-max-frequency = <20000000>;
+		reg = <0>;
+	};
+};
+
+&i2c0 {
+	status = "okay";
+};
+
+&duart0 {
+	status = "okay";
+};
-- 
1.7.1

